include/crm/common/internal.h

```diff
@@ -197,6 +197,15 @@ extern int pcmk__score_green;
 extern int pcmk__score_yellow;
 
 
+/* Error domains for use with g_set_error (from results.c) */
+
+GQuark pcmk_rc_error_quark(void);
+GQuark pcmk_exitc_error_quark(void);
+
+#define PCMK_RC_ERROR       pcmk_rc_error_quark()
+#define PCMK_EXITC_ERROR    pcmk_exitc_error_quark()
```

Comment:
Ah just remembered ... since these are internal, they should all have double underbar

Reply:
Ah right, they weren't originally because I had them in a public header.  Fixed.